src/cockpit/bridge.py

```diff
@@ -157,21 +157,18 @@ def try_to_receive_stderr():
 def setup_logging(debug: bool):
     """Setup our logger with optional filtering of modules if COCKPIT_DEBUG env is set"""
 
-    cfg = {}
-    modules = set(os.getenv('COCKPIT_DEBUG', '').split(','))
-
-    def filter_record(record: logging.LogRecord):
-        if record.name in modules or 'all' in modules:
-            return record
-
-    if debug or modules:
-        cfg['level'] = logging.DEBUG
-    if modules:
-        handler = logging.StreamHandler()
-        handler.addFilter(filter_record)
-        cfg['handlers'] = [handler]
-
-    logging.basicConfig(**cfg)
+    modules = os.getenv('COCKPIT_DEBUG', '')
+    if debug or modules == 'all':
+        logging.basicConfig(level=logging.DEBUG)
+    elif modules:
+        # HACK: Required to make loggers have a handler
+        logging.basicConfig(format='%(name)s-%(levelname)s: %(message)s')
```

Comment:
I think I'd prefer to see the format set in any case, even if debugging is not enabled.

Then for the "all" case I think that could be better handled with `logging.root.setLevellogging.DEBUG)` to make it more symmetric with the per-domain case.